src/views/catalog/CreateFromInstanceTypes/state/hooks/useInstanceTypesAndPreferences.ts

```diff
@@ -10,19 +10,20 @@ type UseInstanceTypeAndPreferences = () => UseInstanceTypeAndPreferencesValues;
 const useInstanceTypesAndPreferences: UseInstanceTypeAndPreferences = () => {
   const [clusterInstanceTypes, clusterITsLoaded, clusterITsLoadError] = useClusterInstanceTypes();
 
-  const [userInstanceTypes, userITsLoaded, userITsLoadError] = useUserInstanceTypes();
+  const [vmInstanceTypes, userITsLoaded, userITsLoadError] = useVirtualMachineInstanceTypes();
 
   const [preferences, preferencesLoaded, preferencesLoadError] = useClusterPreferences();
 
   const loaded = preferencesLoaded && clusterITsLoaded && userITsLoaded;
   const loadError = preferencesLoadError || clusterITsLoadError || userITsLoadError;
 
   return {
+    allInstanceTypes: [...clusterInstanceTypes, ...vmInstanceTypes],
     clusterInstanceTypes,
     loaded: loaded || Boolean(loadError),
     loadError,
     preferences,
-    userInstanceTypes,
+    vmInstanceTypes,
```

Comment:
do we need it? Does some component consume it?

Reply:
Nothing is using it currently. I removed it.